tests/unit/test_parser.py

```diff
@@ -80,3 +80,21 @@ def test_duplicate_line():
     )
     with pytest.raises(IndexError, match=r".*This error is likely from a duplicate line detected.*"):
         compliance.parser_map["cisco_ios"](logging).config_lines  # pylint: disable=expression-not-assigned
+
+
+def test_leading_spaces_config_start():
+    logging = (
+        "! Command: show running-config\n"
+        " 24.1.4\n"
+        "!\n"
+        "no aaa root\n"
+        "!\n"
+        "management api http-commands\n"
+        "   no shutdown\n"
+        "no service interface inactive port-id allocation disabled\n"
+        "transceiver qsfp default-mode 4x10G\n"
+    )
+    with pytest.raises(IndexError, match=r".*Validate the first line does not begin with a space.*"):
+        compliance.parser_map["cisco_ios"](logging).config_lines  # pylint: disable=expression-not-assigned
+        compliance.parser_map["arista_eos"](logging).config_lines  # pylint: disable=expression-not-assigned
```

Comment:
presumably there should be 3 `with pytest.raises` here.

Reply:
agreed

Reply:
probably pytest parameterize would work best